From fa4c79fa41f37835c4f772f893d361fb8f5cce5b Mon Sep 17 00:00:00 2001
From: jerome <jerome.anand@intel.com>
Date: Wed, 27 Feb 2013 14:26:11 +0530
Subject: [PORT FROM MAIN][DO NOT UPSTREAM][Background Music] Disable HDMI support for remote playback

BZ: 63786

Support WIDI background streaming is the only requirement.
HDMI sink is used for testing the solution and can be removed
for the final merge.

Change-Id: I27c9043faf9e9b67a704ed46748f948227bf465a
Orig-Change-Id: I1f3c413348e80f43142a3d99f534ab6c4c7bc939
Signed-off-by: jerome <jerome.anand@intel.com>
Signed-off-by: jerome <jerome.anand@intel.com>
---
 include/system/audio.h |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index f5a3703..6622b83 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -365,11 +365,13 @@ enum {
                                AUDIO_DEVICE_IN_FM_RECORD |
                                AUDIO_DEVICE_IN_DEFAULT),
     AUDIO_DEVICE_IN_ALL_SCO = AUDIO_DEVICE_IN_BLUETOOTH_SCO_HEADSET,
-    //devices not supported for codec offload
+    // devices not supported for codec offload
     AUDIO_DEVICE_OUT_NON_OFFLOAD = (AUDIO_DEVICE_OUT_ALL & ~(AUDIO_DEVICE_OUT_SPEAKER |
                                  AUDIO_DEVICE_OUT_WIRED_HEADSET |
                                  AUDIO_DEVICE_OUT_WIRED_HEADPHONE |
                                  AUDIO_DEVICE_OUT_EARPIECE)),
+    // device supporting remote background playback in addition to foreground playback
+    AUDIO_DEVICE_OUT_REMOTE_BGM_SINK = AUDIO_DEVICE_OUT_WIDI,
 };
 
 typedef uint32_t audio_devices_t;
-- 
1.7.4.1

